docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -85,45 +85,46 @@ set of management operations:
 
 | *view*                    | Defines if a realm administrator can view groups.
 | *manage*                  | Defines if a realm administrator can manage groups.
-| *view-members*            | Defines if a realm administrator can view group members. 
+| *view-members*            | Defines if a realm administrator can view group members.
 | *manage-members*          | Defines if a realm administrator can manage group members.
+| *impersonate-members*     | Defines if a realm administrator can impersonate group members.
 | *manage-membership*       | Defines if a realm administrator can add or remove members from groups.
 |===
 
 [NOTE]
 ====
-- An administrator must have explicit permission assigned for each operation to perform the corresponding action. For example, 
-  assigning only *manage* without *view* will prevent the group from being visible.  
-- The *view-members* and *manage-members* operations apply to the entire group hierarchy. In other words, if an administrator has 
-  permission to manage members of a group, they can also manage members of all its subgroups. This can be prevented by 
+- An administrator must have explicit permission assigned for each operation to perform the corresponding action. For example,
+  assigning only *manage* without *view* will prevent the group from being visible.
+- The *view-members* and *manage-members* operations apply to the entire group hierarchy. In other words, if an administrator has
```

Comment:
@pedroigor `impersonate-members` also evaluates whole group hierarchy, correct?

Reply:
Added a second commit with some updates to the docs. I can send a separate PR if you prefer, but perhaps we can just agree on it here.